conformance/lang/expressions/logical-expressions/logical_or.balt

```diff
@@ -0,0 +1,703 @@
+    A a = true;
+    io:println(true || <boolean>a); // @output true
+    io:println(<boolean>a || true); // @output true
+    io:println(false || <boolean>a); // @output true
+    io:println(<boolean>a || false); // @output true
+    io:println(<boolean>a || <boolean>a); // @output true
+}
+
+Test-Case: output
+Description: Test logical expression using unary expression.
+Labels: logical-expr, unary-expr
+
+function init() {
+    io:println(true || !false); // @output true
+    io:println(!false|| true); // @output true
+    io:println(!true|| false); // @output false
+    io:println(false|| !true); // @output false
+    io:println(!true || !false); // @output true
+}
+
+
+Test-Case: output
+Description: Test logical expression using relational expression.
+Labels: logical-expr, relational-expr
+
+function init() {
+    io:println(true || 4>2); // @output true
+    io:println(4>2 || true); // @output true
+    io:println(false || 4>2); // @output true
+    io:println(4>2 || false); // @output true
+    io:println(4>3 || 4>3); // @output true
+
+    io:println(true || 4<2); // @output true
+    io:println(4<2 || true); // @output true
+    io:println(false || 4<2); // @output false
+    io:println(4<2 || false); // @output false
+    io:println(4<3 || 4<3); // @output false
+
+    io:println(true || 4>=2); // @output true
+    io:println(4>=2 || true); // @output true
+    io:println(false || 4>=2); // @output true
+    io:println(4>=2 || false); // @output true
+    io:println(4>=3 || 4>=3); // @output true
+
+    io:println(true || 4<=2); // @output true
+    io:println(4<=2 || true); // @output true
+    io:println(false || 4<=2); // @output false
+    io:println(4<=2 || false); // @output false
+    io:println(4<=3 || 4<=3); // @output false
+
+    io:println(4<2 || 4>2); // @output true
+    io:println(4>=2 || 4<=3); // @output true
+    io:println(4>2 || 4<=2); // @output true
+    io:println(4>=2 || 4<2); // @output true
+}
+
+Test-Case: output
+Description: Test logical expression using is expression.
+Labels: logical-expr, is-expr
+
+function init() {
+    io:println(true || 2 is int); // @output true
+    io:println(2 is int || true); // @output true
+    io:println(false || 2 is int); // @output true
+    io:println(2 is int || false); // @output trye
+    io:println(2 is int || 2 is int); // @output true
+}
+
+Test-Case: output
+Description: Test logical expression using equality expression.
+Labels: logical-expr, equality-expr
+
+function init() {
+    io:println(true || 2==3); // @output true
+    io:println(3==3 || true); // @output true
+    io:println(false || 3==3); // @output true
+    io:println(2==3 || false); // @output false
+    io:println(2==3 || 3==3); // @output true
+
+    io:println(true || 2!=3); // @output true
+    io:println(3!=3 || true); // @output true
+    io:println(false || 3!=3); // @output false
+    io:println(2!=3 || false); // @output true
+    io:println(2!=3 || 3!=3); // @output false
```

Comment:
`// @output true` since `2 != 3` is true

Reply:
It is marked as `true` right

Reply:
in line 633?